From ad9b6cfb41e48b968cbc0fa1bb1dc309a2990fa1 Mon Sep 17 00:00:00 2001
From: Elliott Sales de Andrade <quantum.analyst@gmail.com>
Date: Wed, 20 May 2026 14:42:10 +0200
Subject: [PATCH 1/2] Don't try to import all of tools

We delete gopls to package it separately, so 'all of tools' is unavailable.

Co-authored-by: Alejandro Sáez <asm@redhat.com>
Signed-off-by: Elliott Sales de Andrade <quantum.analyst@gmail.com>
---
 internal/gcimporter/stdlib_test.go | 9 ---------
 1 file changed, 9 deletions(-)

diff --git a/internal/gcimporter/stdlib_test.go b/internal/gcimporter/stdlib_test.go
index 38488417b..6b853ff55 100644
--- a/internal/gcimporter/stdlib_test.go
+++ b/internal/gcimporter/stdlib_test.go
@@ -9,7 +9,6 @@ import (
 	"fmt"
 	"go/token"
 	"go/types"
-	"runtime"
 	"testing"
 	"unsafe"
 
@@ -36,14 +35,6 @@ func TestStdlib(t *testing.T) {
 	cfg := &packages.Config{Mode: packages.LoadAllSyntax}
 	patterns := []string{"std"}
 	minPkgs := 225 // 'GOOS=plan9 go1.18 list std | wc -l' reports 228; most other platforms have more.
-	switch runtime.GOOS {
-	case "android", "ios":
-		// The go_.*_exec script for mobile builders only copies over the source tree
-		// for the package under test.
-	default:
-		patterns = append(patterns, "golang.org/x/tools/...")
-		minPkgs += 160 // At the time of writing, 'GOOS=plan9 go list ./... | wc -l' reports 188.
-	}
 	pkgs, err := packages.Load(cfg, patterns...)
 	if err != nil {
 		t.Fatalf("failed to load/parse/type-check: %v", err)
-- 
2.54.0

